src/lxc/start.c

```diff
@@ -1549,6 +1613,21 @@ static int lxc_spawn(struct lxc_handler *handler)
 		}
 	}
 
+	if (conf->lxc_shmount.path_host) {
+		if(!conf->lxc_shmount.path_cont) {
```

Comment:
Would probably be nicer if you do a:
```
 if (conf->lxc_shmount.path_host || !conf->lxc_shmount.path_cont)
```

Reply:
I don't think so because the enclosing `if` is bigger than just this check. Or you mean to do the `if (conf->lxc_shmount.path_host)` twice?

Reply:
We usually just try to avoid nesting `if`-conditions too much. Not very pleasant to read usually. :)

Reply:
Yeah, right